app/views/test_executions/_test_info.html.erb

```diff
@@ -11,7 +11,7 @@
   <% end %>
   <strong>HQMF ID: </strong><span><%= task.measure_ids.join(',') %></span><br/>
   <% unless task.product_test.is_a? ChecklistTest %>
-    <strong>CMS ID: </strong><span><%= task.product_test_cms_id %></span><br/>
+    <strong>CMS ID: </strong><span><%= task.product_test_cms_id%></span><span><%= link_to " (eCQM Specification)", "https://ecqi.healthit.gov/ecqm/measures/#{task.product_test_cms_id.sub(/(?<=cms)(\d{1,3})/i) { |s| $1.rjust(3,'0') } }", :target => "_blank", :id => "ecqm-link" %></span><br/>
```

Comment:
Would it be better to move this into the task model and just have it say `task.padded_cms_id` here? This just seems a bit messy for directly in the view. Not necessarily a blocker to merging this, just might be cleaner.